docs/Public_API.rst

```diff
@@ -0,0 +1,948 @@
+API Server Public Rest API
+==========================
+
+Scaling History API
+-------------------
+
+**List scaling history of an application**
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**GET /v1/apps/:guid/scaling\_histories**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Request**
+^^^^^^^^^^^
+
+Route
+'''''
+
+    GET /v1/apps/8d0cee08-23ad-4813-a779-ad8118ea0b91/scaling\_histories
+
+Parameters
+''''''''''
+
++--------------------+-------------------------------------------------------------------------------+---------------------------------------------------------------------+-----------------------+----------------------------------+
+| Name               | Description                                                                   | Valid values                                                        | Required              | Example values                   |
++--------------------+-------------------------------------------------------------------------------+---------------------------------------------------------------------+-----------------------+----------------------------------+
+| guid               | The GUID of the application                                                   |                                                                     | true                  |                                  |
++--------------------+-------------------------------------------------------------------------------+---------------------------------------------------------------------+-----------------------+----------------------------------+
+| start-time         | The start time                                                                | int, the number of nanoseconds elapsed since January 1, 1970 UTC.   | false                 | start-time=1494989539138350432   |
```

Comment:
add start time default when it is not specified, same for end time and other places